assets/src/edit-story/components/panels/design/textStyle/padding.js

```diff
@@ -65,38 +71,52 @@ function PaddingControls({
   pushUpdateForObject,
   pushUpdate,
 }) {
-  const padding = useCommonObjectValue(
-    selectedElements,
+  const displayPadding = useCommonObjectValue(
```

Comment:
Perhaps `displayedPadding`? `displayPadding` sounds like a boolean. WDYT?

Reply:
yea I like it. Added!